.github/workflows/ci_reusable_wf.yml

```diff
@@ -172,7 +171,6 @@ jobs:
         run: |
           cargo clippy --all-targets -- --deny warnings
           cargo clippy --all-targets --all-features -- --deny warnings
-
```

Comment:
My suggestion accidentally deleted the line instead of the spaces. Can you please add a new line here again?